File: apps/institution-api/Models/VwCurrentInstitutionEmail.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace ProviderApi.Models
{
    public partial class VwCurrentInstitutionEmail
    {
        public int InstitutionEmailId { get; set; }
        public int InstitutionId { get; set; }
        public string InstitutionType { get; set; }
        public string InstitutionName { get; set; }
        public string InstitutionCode { get; set; }
        public string EmailUsage { get; set; }
        public string EmailType { get; set; }
        public int? ListSequence { get; set; }
        public string EmailAddress { get; set; }
        public Guid? EmailGuid { get; set; }
    }
}

File: apps/planmp/src/libs/common/models/vw-current-institution-email.ts



export class VwCurrentInstitutionEmail {
    public institutionEmailId: number = 0;
    public institutionId: number = 0;
    public institutionType: string = '';
    public institutionName: string = '';
    public institutionCode: string = '';
    public emailUsage: string = '';
    public emailType: string = '';
    public listSequence: number;
    public emailAddress: string = '';
    public emailGuid: string;

    constructor(init? : Partial<VwCurrentInstitutionEmail>) {
        
        Object.assign(this, init);
    }
}

